theSingularity.js

```diff
@@ -0,0 +1,389 @@
+const readlineSync = require("readline-sync");
```

Comment:
Work on better naming of your commit messages. They should be more frequent and say what they do.